datalake/src/main/resources/application.yml

```diff
@@ -100,4 +100,7 @@ cb:
     disabled: false
 
 notification:
-  urls: http://localhost:3000/notifications
\ No newline at end of file
+  urls: http://localhost:3000/notifications
+
+clusterProxy:
```

Comment:
Note: we need to add cluster proxy URL in CBD and helm-charts as well

Reply:
Good point, I'm looking into it.